rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -339,7 +318,7 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
     val splitRand = rand.splitByte(0)
     /* @bundle+ { x } !(7) -> x!(7)
      */
-    val channel = GString("channel")
+    val channel: Par = GString("channel")
```

Comment:
Maybe add a comment saying `: Par` is required for Scala to resolve the implicit conversions

Reply:
It's a pattern used over and over again when we work with Par-s, so I don't think such a comment is needed